channeldb/channel.go

```diff
@@ -889,6 +889,18 @@ func (c *OpenChannel) String() string {
 	)
 }
 
+// OpeningParty returns the ChannelParty that originally opened this channel.
+func (c *OpenChannel) OpeningParty() lntypes.ChannelParty {
+	c.RLock()
+	defer c.RUnlock()
+
+	if c.IsInitiator {
+		return lntypes.Local
+	} else {
+		return lntypes.Remote
```

Comment:
Can do an early return as the linter notes.

Reply:
It's not really an early return though. Both levels are "equally privileged". I'll change though to please the lint gods.